applications/nexus/nex2ecl.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+   Copyright (C) 2017  Statoil ASA, Norway.
+   The file 'nex2ecl' is part of ERT - Ensemble based Reservoir Tool.
+
+   ERT is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   ERT is distributed in the hope that it will be useful, but WITHOUT ANY
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or
+   FITNESS FOR A PARTICULAR PURPOSE.
+
+   See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+   for more details.
+*/
+
+#include <cctype>
+
+#include <string>
+
+#include <iostream>
+
+#include <ert/nexus/nexus_plot.hpp>
+
+
+void create_ecl_sum(char *file_path) {
+    std::string filename(file_path);
+    auto ending = filename.rfind(".plt");
```

Comment:
I would prefer if the path manipulations were done with the util functions - yes they are quite ugly plain C, but they work - even on Windows:
```C
char * basename;
util_alloc_file_components( file_path , NULL, &basename , NULL);
util_strupr( basename );

ecl_sum_type * ecl_sum = nex::ecl_summary( basename, plt);
ecl_sum_fwrite( ecl_sum );
ecl_sum_free( ecl_sum );
free( basename );
```

Reply:
OK. Fixed now